react/src/hooks/participants/useParticipantsPage.tsx

```diff
@@ -3,9 +3,11 @@ import { useQueryClient } from "react-query";
 import { Participant } from "../../typings";
 import { queryTableData } from "../utils";
 
+export const GET_PARTICPANTS_URL = "/api/participants";
+
 async function fetchParticipants(query: Query<Participant>) {
     // fetch
-    const queryResult = await queryTableData<Participant>(query, "/api/participants");
+    const queryResult = await queryTableData<Participant>(query, GET_PARTICPANTS_URL);
```

Comment:
Typo. I'm also not sure what this change accomplishes — is it to be consistent with the analyses hook?

Reply:
Nevermind, this gets used in components now for the download csv hook

Reply:
Yes -- it's so we can export it and the hook can use it. The hook needs only a url -- the params are supplied by the caller (since they might come from a table or the autocomplete component, we can't totally abstract that piece) and the useQuery hook is built and managed by the useDownloadCsv hook. I put the url in a constant for DRY purposes in case the the url gets changed.